accounts/management/commands/monitor_async_deleted_users.py

```diff
@@ -0,0 +1,127 @@
+#
+# Freesound is (c) MUSIC TECHNOLOGY GROUP, UNIVERSITAT POMPEU FABRA
+#
+# Freesound is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Freesound is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Authors:
+#     See AUTHORS file.
+#
+
+import logging
+
+import requests
+from django.conf import settings
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+
+from accounts.admin import DELETE_SPAMMER_USER_ACTION_NAME, FULL_DELETE_USER_ACTION_NAME, \
+    DELETE_USER_DELETE_SOUNDS_ACTION_NAME, DELETE_USER_KEEP_SOUNDS_ACTION_NAME
+from accounts.models import Profile
+
+logger = logging.getLogger('console')
+
+
+class Command(BaseCommand):
+    help = 'Gets a list of users that should have been asynchronously deleted recently and makes sure that these were' \
+           'indeed deleted. The list of users is taken from Graylog "Delete user request" log messages.'
```

Comment:
I wonder if we can improve this in the future? Our Graylog setup is pretty stable, except for the times when it isn't. Do we think that we might ever have a situation where an account is deleted, gearman is down, and graylog doesn't record the log message?

I think that my solution for ensuring that we don't lose these messages would be to make a new table, DeleteRequest, and then have the gearman job remove the row from this table when completed.

This isn't something that I think needs to be changed in this PR, but we could make a ticket for it to improve later.

Reply:
Addressed in 82194ac4d782e4ad84a29d22fe99ccdb44def351. We now use `DeletionUserRequest` table to check for users that should have been deleted and were not.